src/uct/ib/dc/dc_mlx5.c

```diff
@@ -471,6 +483,8 @@ static ucs_status_t uct_dc_mlx5_iface_create_dci(uct_dc_mlx5_iface_t *iface,
     }
 
     uct_rc_txqp_available_set(&dci->txqp, dci->txwq.bb_max);
+    ++iface->tx.dci_counter;
+    dci->initialized = 1;
```

Comment:
if we have iface->tx.dci_counter, why need also initialized field? we could check if dci_index<dci_counter

Reply:
Not in random policy